types/types.d.ts

```diff
@@ -415,9 +419,12 @@ export interface Page {
    * Emitted when attachment download started. User can access basic file operations on downloaded content via the passed
    * [Download] instance.
    * 
-   * > **NOTE** Browser context **must** be created with the `acceptDownloads` set to `true` when user needs access to the
-   * downloaded content. If `acceptDownloads` is not set or set to `false`, download events are emitted, but the actual
-   * download is not performed and user has no access to the downloaded files.
+   * :::note
```

Comment:
this `:::note` isn't official markdown syntax, so shall we maybe transform it for the types and api.json so editors like VSC can display it correctly?

Reply:
Yeah, we should do it at a codegen stage, was planning on doing it later, but might as well do it right away.